File: pyg-manager-web/src/main/webapp/js/service/seckillGoodsService.js
//服务层
app.service('seckillGoodsService',function($http){
	    	
	//读取秒杀商品商品列表
	this.findKillGoodsList=function(){
		return $http.get('../seckillGoods/findKillGoodsList');		
	}
	
	//修改秒杀商品状态
	this.updateStatus=function(ids,status){
		return $http.get('../seckillGoods/updateStatus?ids='+ids+'&status='+status);		
	}
	
});

File: pyg-manager/pyg-manager-interface/src/main/java/com/pyg/manager/service/SeckillGoodsService.java
package com. pyg.manager.service;
import java.util.List;

import com.pyg.pojo.PageResult;
import com. pyg.pojo.TbSeckillGoods;
import com.pyg.utils.PygResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface SeckillGoodsService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbSeckillGoods> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbSeckillGoods seckillGoods);
	
	
	/**
	 * 修改
	 */
	public void update(TbSeckillGoods seckillGoods);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbSeckillGoods findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long [] ids);


	/**
	 * 修改商家提交的秒杀商品的状态
	 * 
	 */
	public PygResult updateStatus(Long[] ids,String status);

	
}
